.teamcity/patches/projects/_Self.kts

```diff
@@ -0,0 +1,24 @@
+package patches.projects
```

Comment:
missing header file

Reply:
This file was accidentally added by Travis and will be removed.